### ISSUE TITLE
Debugging in "Debugger connect IDE" mode: auto-restart?

### ISSUE THREAD

**Issue by rulatir** [EmmyLua/IntelliJ-EmmyLua#590]
Environment（环境）
--------------
   name   |   version  
----------|----------
 IDEA version         | PhpStorm 2024.1.5
 EmmyLua version | v1.4.14
 OS                        | Linux

What are the steps to reproduce this issue?（重现步骤？）
------------------------------------------------
1. Start a "Debugger connect IDE" launch configuration
2. Start debuggee program that initiates communication with the IDE using emmy protocol
3. Let the debuggee program terminate

What happens?（出现什么问题？）
-------------
The IDE should continue to listen for further connections.

What were you expecting to happen?（期望？）
-----------------------------------------
The launch configuration terminates. In order to debug another run of the program, one must manually restart the launch configuration.

**Comment by CppCXY** [comment]
You're right, but EmmyLua hasn't had any updates for a long time.. I recommend using EmmyLua2.
